zebra/zebra_ns.c

```diff
@@ -123,6 +123,8 @@ void zebra_ns_startup_continue(struct zebra_dplane_ctx *ctx)
 		vlan_read(zns);
 		kernel_read_pbr_rules(zns);
 		kernel_read_tc_qdisc(zns);
+
```

Comment:
Could we have a comment here about how critical this step is?

Reply:
dnoe